Library/Homebrew/test/.codecov.yml

```diff
@@ -1,7 +0,0 @@
-comment: false
-coverage:
-  fixes:
-    - "::Library/Homebrew/"
-  parsers:
-    v1:
-      include_full_missed_files: yes
```

Comment:
It says in the docs that this is deprecated. Not sure if this was fixed in some way.